python/ray/tune/examples/pbt_example.py

```diff
@@ -86,26 +86,10 @@ def reset_config(self, new_config):
     parser.add_argument(
         "--smoke-test", action="store_true", help="Finish quickly for testing"
     )
-    parser.add_argument(
```

Comment:
I would keep the --cluster flag just in case it is used in nightly tests

Reply:
I checked around, seems like nobody is using this outside.

Also, even if they do pass it, the parser will catch all extra flags and not error. `ray.init("auto") == ray.init()`, so the behavior will be the same.